### PR TITLE
feat(deps): upgrade deps

### LINKED ISSUE #826: 升级依赖版本
https://david-dm.org/ant-design/ant-design-mobile

### PR DESCRIPTION
**Pull request by paranoidjk**
close #826

First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,20 +41,20 @@
     "rc-collapse": "^1.6.11",
     "rc-dialog": "^6.5.5",
     "rc-drawer": "^0.4.9",
-    "rc-input-number": "^2.8.8",
+    "rc-input-number": "^3.0.4",
```
Comment (afc163) [inline package.json]: ~

Comment (paranoidjk) [inline package.json]: any particular reason? why this need  to lock the minor version while other rc-componnet both are just lock major version

Comment (afc163) [inline package.json]: We don't expect any new features in `antd` or `antd-mobile` without upgrading their's minor version, which could be bringed from react-component.

New features should be released within  minor version of `antd` or `antd-mobile`.

Comment (paranoidjk) [inline package.json]: so, all rc-components dependencies of `ant-design-mobile` should use `~` to lock minor version,  just like what  `ant-design` did: https://github.com/ant-design/ant-design/blob/master/package.json#L43

cc @warmhug @silentcloud

Comment (paranoidjk) [inline package.json]: done https://github.com/ant-design/ant-design-mobile/commit/e51c204c06ed06b99485d9c68ade114e304ac5f8

Comment (paranoidjk) [inline package.json]: cc @silentcloud 有啥问题在这回复？

**Pull request merged**